lib/ain-evm/src/evm.rs

```diff
@@ -143,6 +154,22 @@ impl EVMServices {
 
         let mut executor = AinExecutor::new(&mut backend);
 
+        if current_block_number == U256::zero() {
+            let CounterContractInfo {
+                address,
+                storage,
+                bytecode,
+            } = EVMServices::counter_contract().unwrap();
```

Comment:
We should use proper error handling in ain-contracts lib functions and propagate them properly here.